pkg/anchor/writer/writer.go

```diff
@@ -225,9 +238,60 @@ func (c *Writer) handle(vc *verifiable.Credential) {
 		return
 	}
 
-	// TODO: announce anchor to followers and node observer (if running in observer node)
-
 	c.anchorCh <- []string{cid}
+
+	// announce anchor credential activity to followers
+	err = c.postActivity(vc, cid)
+	if err != nil {
+		logger.Warnf("failed to post new activity for cid[%s]: %s", cid, err.Error())
+
+		return
+	}
+}
+
+// postActivity creates and posts new activity to activity pub.
+func (c *Writer) postActivity(vc *verifiable.Credential, cid string) error { //nolint: interfacer
+	cidURL, err := url.Parse("https://orb.domain1.com/cas/" + cid)
```

Comment:
Hard-coded URL?

Reply:
We should discuss CAS client and handling URLs

Reply:
fixed, added cas IRI